lib/Conversation/ConversationFooter.js

```diff
@@ -1,59 +1,14 @@
-import React from 'react';
-import { arrayOf, bool, func, string, shape } from 'prop-types';
-import CommentForm from './CommentForm';
+import React, { Fragment } from 'react';
+import { node } from 'prop-types';
 
-export function ConversationFooter({
-  id,
-  addComment,
-  user,
-  onCancel,
-  onCommentChange,
-  focusOnMount,
-  placeholder,
-  onRowCountChange,
-  users
-}) {
-  return (
-    <div className="conversation__foot">
-      <CommentForm
-        id={id}
-        onSubmit={addComment}
-        author={user}
-        onCancel={onCancel}
-        onCommentChange={onCommentChange}
-        focusOnMount={focusOnMount}
-        placeholder={placeholder}
-        onRowCountChange={onRowCountChange}
-        users={users}
-      />
-    </div>
-  );
+export function ConversationFooter({ children }) {
+  return <div className="p-2">{children}</div>;
```

Comment:
So the footer used to house the new comment form, but now it just renders children. Which probably will still be the new comment form